### ISSUE TITLE
Flows - can't pass flow output to another flow

### ISSUE THREAD

**Issue by dkamali-mc** [directus/directus#14069]
### Describe the Bug

Not able to pass Flow data from one flow to another 

Data always return an empty object `{}`

### To Reproduce

-  create two flows A and B (note the UUID for flow B)  
-  configure flow A to trigger/invoke flow B and use the default payload) 
```
{
  "user": "{{ $accountability.user }}",
  "data": "{{ $last }}"
}
```

### What version of Directus are you using?

v9.12.2

### What version of Node.js are you using?

Nodejs - PaaS

### What database are you using?

Postgres  - PaaS

### What browser are you using?

Chrome

### How are you deploying Directus?

PaaS

**Comment by br41nslug** [comment]
This is happenning because ```json
{
  "user": "{{ $accountability.user }}",
  "data": "{{ $last }}"
}
```
ends up with unescaped json in the `data` string which errors when the next flow tries to parse this data because it is invalid json.

**Comment by emilyf** [comment]
Believe this is also my issue after search all around but apologies if i'm still in wrong place, originally posted at #14971 14971

on enterprise cloud 9.18.1

Using Flows as described to run a POST call on external webhook I'm finding no matter how I clean the data I end up with this same axios error as reported above. In the screenshots below you can see my output is formatted cleanly all the way up until the final webhook payload when it starts escaping characters in the fields. I was searching around for this starting on https://github.com/directus/directus/issues/14265 and ended up here. Below are screenshots; I'm on enterprise cloud so I don't think I have ability to directly configure axios (assuming that's where my issue is). 

Note: I have triggered the same response body through postman with same headers, same auth, same json etc and it comes back successful.

My flow is triggered by another flow, here's the payload I have coming out of that which is looking how I want.

<img width="503" alt="trigger_payload" src="https://user-images.githubusercontent.com/1789807/196298717-6fa08973-6295-4e9f-aa1f-c00e1f0c2343.png">

From there i'm just triggering a webhook and my output here is still holding on....

<img width="625" alt="webhook request options" src="https://user-images.githubusercontent.com/1789807/196299089-a2d1c547-b3cd-4d12-8666-02fbf47dfa45.png">

Then it hits the payload for the webhook and I get the 400 Axios error. 

<img width="647" alt="payload_error" src="https://user-images.githubusercontent.com/1789807/196299211-0fbd080e-3553-4ec6-a6bb-634d1d0e4782.png">

**Comment by ConnorSimply** [comment]
Does this PR/Issue https://github.com/directus/directus/pull/15946 sound like what is happening? If not can you share how you have your flow set up for the one that isn't working?

**Comment by emilyf** [comment]
@ConnorSimply Yes! I just happened upon that PR from reading https://github.com/directus/directus/discussions/15945 and further attempting every which way to adjust. I'm waiting with baited breath for it to be released!

**Comment by ConnorSimply** [comment]
I had something very similar happening to me, have you tried recreating the Webhook operation and using the raw editor from the get go and making sure you are not wrapping the mustache template in quotes?

**Comment by rijkvanzanten** [comment]
Linear: [ENG-243](https://linear.app/directus/issue/ENG-243/flows-cant-pass-flow-output-to-another-flow)

**Comment by nickrum** [comment]
The original issue has been fixed by #16180. The issue described by @br41nslug related to unescaped json has been fixed by #15946.
